<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registration</class>
 <widget class="QMainWindow" name="registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>451</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 2px solid rgb(16, 16, 16);
    border-radius: 15px;
    margin-top: 10px; 
    background-color:rgb(232, 232, 232);
    padding: 5px;
 
}

</string>
    </property>
    <property name="title">
     <string>Register</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Create your Account Here</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Phone No.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Email ID</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Grade</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Reg No.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEdit_regname">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
    border: 2px solid rgb(16, 16, 16);
    border-radius: 10px; 
    background-color: rgb(255, 255, 255,0);
    min-width: 8px; 
    min-height: 8px;
    padding: 5px;
    margin: 0px;
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_regphone">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
    border: 2px solid rgb(16, 16, 16);
    border-radius: 10px; 
    background-color: rgb(255, 255, 255,0);
    min-width: 8px; 
    min-height: 8px;
    padding: 5px;
    margin: 0px;
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_regemail">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
    border: 2px solid rgb(16, 16, 16);
    border-radius: 10px; 
    background-color: rgb(255, 255, 255,0);
    min-width: 8px; 
    min-height: 8px;
    padding: 5px;
    margin: 0px;
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_regpassword">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
    border: 2px solid rgb(16, 16, 16);
    border-radius: 10px; 
    background-color: rgb(255, 255, 255,0);
    min-width: 8px; 
    min-height: 8px;
    padding: 5px;
    margin: 0px;
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_grade">
             <property name="styleSheet">
              <string notr="true">QComboBox{
    border: 2px solid rgb(16, 16, 16);
    border-radius: 10px; 
    background-color:rgb(255, 255, 255,0);
    min-width: 9px; 
    min-height: 9px; 
    padding: 5px; 
    margin: 0px; 
}
</string>
             </property>
             <item>
              <property name="text">
               <string>1st Sem</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2nd Sem</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3rd Sem</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4th Sem</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5th Sem</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6th Sem</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7th Sem</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8th Sem</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_regno">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
    border: 2px solid rgb(16, 16, 16);
    border-radius: 10px; 
    background-color: rgb(255, 255, 255,0);
    min-width: 8px; 
    min-height: 8px;
    padding: 5px;
    margin: 0px;
}
</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_role">
         <property name="styleSheet">
          <string notr="true">QComboBox{
    border: 2px solid rgb(16, 16, 16);
    border-radius: 10px; 
    background-color:rgb(255, 255, 255,0);
    min-width: 9px; 
    min-height: 9px; 
    padding: 5px; 
    margin: 0px; 
}
</string>
         </property>
         <item>
          <property name="text">
           <string>Student</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Teacher</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_reg">
         <property name="text">
          <string>I agree to terms and conditions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_register">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid rgb(16, 16, 16);
    border-radius: 10px; 
    background-color:rgb(255, 255, 255,0);
    min-width: 9px; 
    min-height: 9px; 
    padding: 5px; 
    margin: 0px; 
}
</string>
         </property>
         <property name="text">
          <string>Register </string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_loginpage">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    border: 2px solid rgb(16, 16, 16);
    border-radius: 10px; 
    background-color:rgb(255, 255, 255,0);
    min-width: 9px; 
    min-height: 9px; 
    padding: 5px; 
    margin: 0px; 
}
</string>
           </property>
           <property name="text">
            <string>Back to Login Page</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_status2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>[+]Status </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
